iroh/src/docs_engine.rs

```diff
@@ -95,13 +103,24 @@ impl Engine {
             .instrument(error_span!("sync", %me)),
         );
 
-        Self {
+        let default_author = match DefaultAuthor::load(default_author_storage, &sync).await {
+            Ok(author) => author,
+            Err(err) => {
```

Comment:
why exit in this case, instead of generating a new one?

Reply:
This only returns an error if parsing the default author file failed. If the default author file is non-existent, a default author is created already without returning an error, see line 337ff.